app/views/layouts/application.html.erb

```diff
@@ -6,9 +6,13 @@
     <%= stylesheet_link_tag    'application', media: 'all',
```

Comment:
- change all `single quote => double quote` in this file
- Thừa khoảng trống:=> `stylesheet_link_tag "application"`